spec/datetimeformat.html

```diff
@@ -63,6 +63,11 @@ <h1>Abstract Operations For DateTimeFormat Objects</h1>
           <td>`"timeZoneName"`</td>
           <td>`"short"`, `"long"`</td>
         </tr>
+        <tr>
+          <td>[[FractionalSecondDigits]]</td>
+          <td>`"fractionalSecondDigits"`</td>
+          <td>`0`, `1`, `2`, `3`</td>
```

Comment:
@anba  Do you think I should change this to <td>`1`, `2`, `3`</td> instead?

Reply:
yes, that sounds correct.